### PR TITLE
Fix home page

### PR DESCRIPTION
**Pull request by thepatrickniyo**
### Bug description
- Fix  common theme.
- Added new logo.
- Fix horizontal alignement of home page content
- Proper import of  Header and Footer components.

### Root cause
- Default material UI background.
- Login and Signup bottom contents was hidden by Footer

### Solution description
- A new logo was designed.
- White them was added globally to override material default theme.
- Header and Footer components were imported in other pages.

### What are the relevant Pivotal Tracker stories?
- [#176048315](https://www.pivotaltracker.com/n/projects/2476677)
- [#176037767](https://www.pivotaltracker.com/n/projects/2476677)
- [#176038228](https://www.pivotaltracker.com/n/projects/2476677)

### Screenshots (if appropriate)
image

### Questions
- N/A

### CODE REVIEW

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]
Good job! But here are few thing to consider.
1. Since  you changed the `general theme` from  `#fafafa` to`#ffffff`you can also change the background of the **Pattern** of the `Signup page`.
2. Another thing is that could you please remove the scroll behavior on the `Login page`

**Comment by malaba6** [comment]
Good job Patrick. However there are a few changes you'll be required to make:
1.


2.

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ const useStyles = makeStyles((theme) => ({
         height: 70,
         background: '#FFFFFF',
         alignItems: 'center',
-        position: 'fixed',
-        bottom: 0,
+        // position: 'fixed',
```
Comment (malaba6) [inline src/components/Footer/index.js]: Remove these commented out lines

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -17,8 +17,8 @@ const useStyles = makeStyles((theme) => ({
...
+        // bottom: 0,
```
Comment (malaba6) [inline src/components/Footer/index.js]: And here!

**Review by malaba6 — CHANGES_REQUESTED** [review]

**Comment by thepatrickniyo** [comment]
> Good job Patrick. However there are a few changes you'll be required to make:
> 1.
> 
> 
> 
>

**Comment by thepatrickniyo** [comment]
Dear @SimonImanigirimpuhwe 
Your issue 1, was solved successfully.
Issue 2, was not solved since contents are not fitting in the screen except for mac. but I kept consistance between two pages signup and login.

**Comment by thepatrickniyo** [comment]
Dear @Malaba6,
Your issues where worked on and now they are resolved successfully.
About the navigation menu items spacing and login button border radius will be found in the PR for @SamuelSabasNayo since he worked on it.

**Review by SamuelSabasNayo — APPROVED** [review]

**Review by marcelniyi — APPROVED** [review]

**Comment by SamuelSabasNayo** [comment]
We would like to ask our managers to merge our PR for us to proceed smoothly.

**Pull request merged**